pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -456,3 +459,19 @@ func generateOwnershipUniqueKeys(owner hivev1.MetaRuntimeObject) []*controllerut
 		},
 	}
 }
+
+// deleteInstallJob deletes the install job of a successful provision
+func (r *ReconcileClusterProvision) deleteInstallJob(provision *hivev1.ClusterProvision, pLog log.FieldLogger) (reconcile.Result, error) {
+	pLog.Info("deleting successful install job")
```

Comment:
This should be debug not info. This will write on every reconcile of a completed provision.

Reply:
Instead do we want to run `deleteInstallJob` only if JobRef is present? If we are clearing JobRef than we don't have to run this code on every reconcile

Reply:
Yes. If we are only running this until the job reference is removed, then Info is a good log level.